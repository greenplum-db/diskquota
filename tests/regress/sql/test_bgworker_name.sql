-- create a database with non-ascii characters
CREATE DATABASE 数据库1;

\c 数据库1

CREATE EXTENSION diskquota;
SELECT diskquota.wait_for_worker_new_epoch();

\! ps -ef | grep postgres | grep "\[diskquota] 数据库1" | grep -v grep | wc -l

DROP EXTENSION diskquota;
\c contrib_regression
DROP DATABASE 数据库1;