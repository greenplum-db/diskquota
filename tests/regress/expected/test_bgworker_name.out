-- create a database with non-ascii characters
CREATE DATABASE 数据库1;
\c 数据库1
CREATE EXTENSION diskquota;
SELECT diskquota.wait_for_worker_new_epoch();
 wait_for_worker_new_epoch 
---------------------------
 t
(1 row)

\! ps -ef | grep postgres | grep "\[diskquota] 数据库1" | grep -v grep | wc -l
0
DROP EXTENSION diskquota;
\c contrib_regression
DROP DATABASE 数据库1;
